signup/static/js/djaodjin-signup-vue.js

```diff
@@ -247,3 +281,31 @@ var app = new Vue({
     },
 })
 }
+
+if($('#update-password-container').length > 0){
+var app = new Vue({
+    el: "#update-password-container",
+    data: {
+        password: '',
+        password_confirmation: '',
```

Comment:
Why use `password_confirmation` when everywhere else we use `new_password2`?

Reply:
I haven't noticed that